core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala

```diff
@@ -29,9 +29,9 @@ abstract class Component[T](val period: Time) {
     * @param controller the new controller to use
     */
   def setController(controller: Stream[T]): Unit = {
-    if (controller.expectedPeriodicity == NonPeriodic) {
-      throw new IllegalArgumentException("Controller must be periodic")
-    }
+    //if (controller.expectedPeriodicity == NonPeriodic) {
```

Comment:
Whoa this is a pretty serious change to be made. This needs to be re-enabled and whichever controller was non-periodic needs to be made periodic.

Reply:
Oh, this was a change I playing around with during the work session that I need to remove